utils/plot.py

```diff
@@ -37,6 +42,35 @@ def plot_dataloader(dataloader: DataLoader, n_images: int = 1):
         display(im, metadata={'width': '100%'})
 
 
+def plot_model_comparison(results_df: pd.DataFrame, fig_size: Tuple[int, int] = (15, 7)):
+    """
+    expects a dataframe with the following columns: [run_id, MSE, PSNR, SSIM]
+    """
+
+    x = np.arange(3)  # the label locations
+    width = 0.8 / len(results_df)  # the width of the bars
+    multiplier = 0
+    fig, ax = plt.subplots(figsize=fig_size)
+    for index, row in results_df.iterrows():
+        a = [np.round(row['MSE'], 5), np.round(row['PSNR'], 2), np.round(row['SSIM'], 3)]
+        measurement = pd.Series(a)
+        attribute = row['run_id']
+        offset = width * multiplier
+        rects = ax.bar(x + offset, measurement, width, label=attribute)
+        ax.bar_label(rects, padding=3)
+        multiplier += 1
+
+    ax.set_ylabel('Value')
```

Comment:
maybe remove y-axis labels and values as the range is simply too big that it would make sense